.github/dependabot.yml

```diff
@@ -1,11 +1,15 @@
 version: 2
 
 updates:
+  # npm should be used for npm and yarn
   - package-ecosystem: npm
     open-pull-requests-limit: 10
     directory: "/"
     schedule:
       interval: "weekly"
       day: "sunday"
+    commit-message:
+      # we need to disable netlify as we don't have enough builds for all dependency updates
+      prefix: "[skip ci]"
```

Comment:
let's use the `[skip netlify]` which is a valid alternative and makes it much clearer that we only target netlify.

> Sometimes, you may want to push commits to your production branch without triggering a deploy on Netlify. 
> To do this, add [skip ci] or [skip netlify] anywhere in the Git commit message.

https://docs.netlify.com/site-deploys/manage-deploys/